src/__fish_command_not_found_handler.fish

```diff
@@ -0,0 +1,12 @@
+function __fish_command_not_found_handler --on-event fish_command_not_found
+	if test -x "/usr/bin/cnf-lookup"
+		cnf-lookup -c -- $argv[1]
+		if test $status -ne 0
+			__fish_default_command_not_found_handler $argv[1]
+		end
+	else
+		__fish_default_command_not_found_handler $argv[1]
+	end
+	# return 127 # This is actually unnecessary; fish handles it
```

Comment:
Remove the comment. It adds no value.

Reply:
My intent in leaving that there was so that if someone looked at it in the future and compared it to the bash functions, they wouldn't think that it was a mistake that `return 127` wasn't included.

Would
```fish
# do not return 127 because fish does that automatically after this function is called
```
be a better fit?

Reply:
# no need to return an error code for fish